ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -226,60 +152,91 @@ The list displays all workflow versions of the selected workflow. Depending on t
+    When clicking a transition in the workflow editor, it becomes highlighted and the corresponding element settings are displayed in the settings sidebar.
 
-    ![Settings side bar](../../Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings side bar]")
+    ![Settings sidebar](../../Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings sidebar]")
 
-    The transition settings side bar is located to the right of the workspace and allows to define the transition settings. The following fields are displayed:
+    The transition settings sidebar is located to the right of the workspace and allows to define the transition settings. The following fields are displayed:
 
     - *Key*  
-        Transition key. This field is read-only. The key is required for API access and must be unique within the workflow version.
+        Transition key. This field is read-only. The key is required for API access and must be unique within the workflow version.<!--- Was ist damit gemeint? und wenn ich eine transition mehrfach einfüge?...-->
 
     - *Label*  
-        Transition label defined by the user. It can be used to specify or simplify the transition name. When editing the transition label, it is changed both at the top of the settings side bar and in the transition rectangle in the diagram.
+        Transition label defined by the user. It can be used to specify or simplify the transition name. When editing the transition label, it is changed both at the top of the settings sidebar and in the transition rectangle in the diagram.
 
     - *Queue type*  
-        Click the drop-down list to select the applicable queue type.   
+        Click the drop-down list to select the applicable queue type, see [Queue types](05_QueueTypes.md).
+
+    - *Priority*   
+        The *Priority* field is used in high performance scenarios. You can use it to speed up a critical path when there are so many actions to be processed that they cannot be processed in parallel, and they pile up in the backlog. You do this by prioritizing the actions that are in the critical path. For example, a critical path might be when an order has been received and needs to be sent to shipping, while other actions such as sending emails are not as time-critical. Note: Priority 10 increases the processing probability by a factor of 10 over priority 1, and priority 8 increases the processing probability by a factor of 4 over priority 2.
+
+    - *Max tries*   
+        Number of times the action execution should be retried after an error has occurred. This setting depends on the transition and is usually preconfigured. If you change this setting to a value greater than 1, you can select a retry time strategy from a drop-down list. Depending on this retry time strategy, the value in the *Strategy config value* field is used to perform the number of retries specified in the *Max tries* field. The following retry time strategies are available:
```

Comment:
Ich musste ein paar Mal lesen und im UI gucken, da "Retry time strategy" und "Strategy config value" nicht angezeigt werden, wenn auf 1 gesetzt. Evtl. ausführlicher beschreiben? 

Z.B. If you change this setting to a value greater than 1, the "Retry time strategy" drop-down list is displayed. In this drop-down list, you can select... When you select a retry time strategy, the "Strategy config value" drop-down list is displayed. In the "Strategy config value", you can set the number...